### PR TITLE
Added types for blobshape

### PR DESCRIPTION
**Pull request by oyedejioyewole**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/blobshape/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Type definitions for blobshape 1.0
+// Project: https://github.com/lokesh-coder/blobshape#readme
+// Definitions by: OyewoleOyedeji <https://github.com/OyewoleOyedeji>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface BlobshapeParameters {
+    size?: number;
+    growth?: number;
+    edges?: number;
+    seed?: null;
```
Comment (peterblazejewicz) [inline types/blobshape/index.d.ts:10]: how this can be null? do meant `@default` values?

File: types/blobshape/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+export default function blobshape({ size, growth, edges, seed }?: BlobshapeParameters): {
```
Comment (peterblazejewicz) [inline types/blobshape/index.d.ts:13]: you've probably need CommonJS export pattern for this module:
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

so probably something along:
```ts
declare function generator({ size, growth, edges, seed }?: generator.BlobshapeParameters): {
    path: string;
    seedValue: number;
};

declare namespace generator {
    interface BlobshapeParameters {
        size?: number;
        growth?: number;
        edges?: number;
        seed?: null;
    }
}

export = generator;
```

Comment (OyewoleOyedeji) [reply types/blobshape/index.d.ts:13]: The package is an ES module

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@OyewoleOyedeji thanks!

**Comment by OyewoleOyedeji** [comment]
Ready to merge

**Pull request merged**